web/src/app.js

```diff
@@ -198,6 +196,33 @@ konnektApp.factory('UserService', ['HttpService', '$window', function (HttpServi
 }]);
 
 
+konnektApp.factory('DataHandling', ['HttpService', function (HttpService) {
```

Comment:
what kind of data? as I see it's all about contacts, let's call it something like that. data is way too broad